src/basic/efivars.c

```diff
@@ -114,6 +114,9 @@ int efi_get_variable(
                         return log_debug_errno(SYNTHETIC_ERRNO(EBUSY), "Reading EFI variable '%s' failed even after %u tries, giving up.", p, try);
                 if (try >= EFI_N_RETRIES_NO_DELAY)
                         (void) usleep_safe(EFI_RETRY_DELAY);
+
+                /* Start from the beginning */
+                (void) lseek(fd, SEEK_SET, 0);
```

Comment:
the whence and offset value params are inverted (this seems to be borked at multiple places in our codebase...